.ci/Jenkinsfile

```diff
@@ -178,6 +179,33 @@ pipeline {
           githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
         }
       }
+      stage('Release') {
+        agent { label 'linux && immutable' }
+        options { skipDefaultCheckout() }
+        environment {
+          HOME = "${env.WORKSPACE}"
+          PATH = "${env.WORKSPACE}/bin:${env.PATH}"
+        }
+        when {
+          beforeAgent true
+          anyOf {
+            tag pattern: 'v\\d+.*', comparator: 'REGEXP'
+            expression { return params.Run_As_Master_Branch }
+          }
+        }
+        steps {
+          withGithubNotify(context: 'Release') {
+            deleteDir()
+            unstash 'source'
+            script {
+              rubygemsLogin.withApi(secret: RUBYGEM) {
+                sh 'rake release'
+                sh 'gem pull x.y'
```

Comment:
would this check that the release went through by downloading it immediately afterwards?

Reply:
It's not required, so I just removed :)